packages/ubus-tmate/etc/tmate/tmate.conf

```diff
@@ -0,0 +1,5 @@
+# in this source-file we expect to have the server connection info, don't remove. If server config doesn't exist (in this or that file), will use default tmate.io servers
```

Comment:
Why is this file not under /files ?
Is it a mistake?

Reply:
it is indeed! thanks for rising that!